common/searchattribute/defs.go

```diff
@@ -70,7 +71,8 @@ const (
 	// Used by scheduler workflow.
 	TemporalSchedulePaused = "TemporalSchedulePaused"
 
-	ReservedPrefix = "Temporal"
+	ScheduleWorkflowIDPrefix = "temporal-sys-scheduler:"
```

Comment:
`common/primitives` makes more sense.. this value has nothing much to do with `searchttribute`